Resources/Maps/Shuttles/Mushroom.yml

```diff
@@ -0,0 +1,5827 @@
+        - 0
+      type: EntityStorage
+    - containers:
+        entity_storage: !type:Container
+          showEnts: False
+          occludes: True
+          ents:
+          - 395
+          - 394
+          - 397
+          - 396
+      type: ContainerContainer
+- proto: ClothingBeltMilitaryWebbing
+  entities:
+  - uid: 862
+    components:
+    - pos: -5.5,-1.5
+      parent: 1
+      type: Transform
+- proto: ClothingHandsGlovesCombat
+  entities:
+  - uid: 865
+    components:
+    - pos: -5.5,-1.5
+      parent: 1
+      type: Transform
+- proto: ClothingHeadHatHoshat
+  entities:
+  - uid: 858
+    components:
+    - desc: Yeehaw
+      name: Bounty Hunter Hat
+      type: MetaData
+    - pos: -5.5,-1.5
+      parent: 1
+      type: Transform
+    - group: ""
+      type: NameIdentifier
+- proto: ClothingMaskGasMerc
+  entities:
+  - uid: 860
+    components:
+    - pos: -5.5,-1.5
+      parent: 1
+      type: Transform
+- proto: ClothingMaskMuzzle
+  entities:
+  - uid: 394
+    components:
+    - flags: InContainer
+      type: MetaData
+    - parent: 393
+      type: Transform
+    - canCollide: False
+      type: Physics
+    - type: InsideEntityStorage
+- proto: ClothingOuterCoatHoSTrench
+  entities:
+  - uid: 852
+    components:
+    - name: Bounty Hunter coat
+      type: MetaData
+    - pos: -5.5,-1.5
+      parent: 1
+      type: Transform
+- proto: ClothingOuterHardsuitBasic
+  entities:
+  - uid: 856
+    components:
+    - pos: -5.5,-1.5
+      parent: 1
+      type: Transform
+    - group:
+      - hoverMessage: ""
+        contextText: verb-examine-group-other
+        icon: /Textures/Interface/examine-star.png
+        components:
+        - Armor
+        - ClothingSpeedModifier
+        entries:
+        - message: This decreases your speed by [color=yellow]20%[/color].
+          priority: 0
+          component: ClothingSpeedModifier
+        - message: >-
+            It provides the following protection:
+
+            - [color=yellow]Blunt[/color] damage reduced by [color=lightblue]10%[/color].
+
+            - [color=yellow]Slash[/color] damage reduced by [color=lightblue]10%[/color].
+
+            - [color=yellow]Piercing[/color] damage reduced by [color=lightblue]10%[/color].
+
+            - [color=yellow]Caustic[/color] damage reduced by [color=lightblue]10%[/color].
+
+            - [color=orange]Explosion[/color] damage reduced by [color=lightblue]10%[/color].
+          priority: 0
+          component: Armor
+        title: null
+      type: GroupExamine
+- proto: ClothingOuterStraightjacket
+  entities:
+  - uid: 395
+    components:
+    - flags: InContainer
+      type: MetaData
+    - parent: 393
+      type: Transform
+    - canCollide: False
+      type: Physics
+    - type: InsideEntityStorage
+- proto: ClothingShoesBootsCombatFilled
+  entities:
+  - uid: 863
+    components:
+    - pos: -5.5,-1.5
+      parent: 1
+      type: Transform
+- proto: ClothingUniformJumpskirtDetective
```

Comment:
Detective gear?

Reply:
the hard worn suit is an jumpsuit that you can buy from the repdrobe that all civils have access too and its on the sprinter ship another Expedition ship

Reply:
these has been corrected